sample/manifest.yml

```diff
@@ -9,3 +9,5 @@ applications:
   path: target/logging-sample-app-2.1.1.war
   env:
     RANDOM_SLEEP: true
+# Set LogRemoteIP: true to activate logging of remote IPs
+    LogRemoteIP: false
```

Comment:
It's not that critical, but I would favor a consistent naming scheme for environment variables. 
In that sense, I would rather go for `LOG_REMOTE_IP`.

Reply:
Done.